internal/grpc/constants.go

```diff
@@ -0,0 +1,25 @@
+package grpc
+
+import "time"
+
+type contextKey string
+
+const (
+	ClientCNKey = contextKey("clientCN")
+
+	// In a real environment these would be set via environment variables and not be hardcoded
+	// we would want to use a config package to manage these values and could load these in from env vars and utilize
+	// https://github.com/kelseyhightower/envconfig
+
+	ServerCertName = "server"
+	CACertFileName = "ca.crt"
```

Comment:
Given that certs holds this information, should we move these constants there as well?

Reply:
Sure, good suggestion